sql/src/main/java/io/crate/operation/join/NestedLoopOperation.java

```diff
@@ -307,7 +307,7 @@ private void killBoth(Throwable throwable) {
 
         private volatile Row lastRow = null; // TODO: volatile is only required for first access
         private RepeatHandle repeatHandle;
-        private boolean wakeupRequired = true;
+        private boolean isPaused = true;
```

Comment:
The value is assigned in setNextRow() and pauseProcessed() which I guessed could be called concurrently but if that's not the case...